src/mds/MDSRank.h

```diff
@@ -153,6 +154,7 @@ class MDSRank {
     MDLog        *mdlog;
     MDBalancer   *balancer;
     ScrubStack   *scrubstack;
+    DamageTable  damage_table;
```

Comment:
RAII -- the pointers shouldn't really be pointers, I'd like to refactor them too at some stage.  We already have e.g. SessionMap here that's not a pointer.